config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get "dev_contacts/new"
+  get "dev_contacts/create"
```

Comment:
I might have missed something but do we need these? `resources :dev_contacts` go to the same actions right?

Reply:
You are very right. I don't know why I did that. Thanks for catching it!